app/controllers/avo/base_controller.rb

```diff
@@ -397,6 +397,7 @@ def create_success_action
 
     def create_fail_action
       respond_to do |format|
+        flash.now[:warning] = reupload_files_message if blobs_present?
```

Comment:
Let's not do this. They should figure it out from the form. It adds extra complexity.

Reply:
Agree